platforms/hyperledger-besu/configuration/roles/create/crypto/orion/tasks/main.yaml

```diff
@@ -0,0 +1,75 @@
+# This tasks checks the crypto material in the vault
+- name: Check for the crypto material in the vault
+  include_tasks: check_vault.yaml
+  vars:
+    component_ns: "{{ org.name }}-bes"
```

Comment:
bes or besu? Stick to one and update all code.

Reply:
it will be bes, will create a bug and fix himanshu's code with -besu